### PR TITLE
Add download functionality

### PR DESCRIPTION
**Pull request by anned20**
#8

### CODE REVIEW

**Comment by daudmalik06** [comment]
@anned20 awesome ,
wouldn't be better if we don't pass arguments to download method , may be
you  can make 2 more methods , 
`downloadPdf `,
` downloadScreenShot`
and those call 
`$this->getPdf()`
`$this->getScreenShot()`
and then download the produced files ?

the idea is user just do this way

`$chrome->downloadScreenShot($optionalName);`

`$chrome->downloadPdf($optionalName);`

optional name would be the name of file he want to see while downloading ,
what do you say ???

and also please put tests for your commits as in [contribution guide](https://github.com/daudmalik06/PhpChromeToPdf/blob/master/CONTRIBUTING.md) .

**Comment by anned20** [comment]
@daudmalik06 I updated the pull request.

File: src/Chrome.php [hunk 1]
```diff
@@ -185,6 +211,30 @@ public function setOutputDirectory($directory=null)
...
+     * @throws \Exception
+     */
+    private function download($path)
+    {
+        if (!file_exists($path)) {
+            throw new \Exception($path.' does not exist or is not readable');
+        }
+
+		if (PHP_SAPI === 'cli') {
+			throw new \Exception('Cannot download from CLI');
```
Comment (dangenendt) [inline src/Chrome.php]: one indentation too much

**Comment by daudmalik06** [comment]
hello @anned20  can you please check your example , it's not working and also your tests are not correct , 
if you need help you can pm me daudmalik06@gmail.com
, 
please fix these issue and update the pull request ,

**Pull request closed without merging**